new/src/components/ScenarioOverview/ScenarioCollectionHead.tsx

```diff
@@ -156,81 +132,97 @@ function ScenarioTitles(props: { headers: Headers }) {
+    onBookmarkButtonClick: () => void;
+}
+
+function ScenarioActionButtons({
+    onCollapseButtonClick,
+    onExpandButtonClick,
+    onPrintButtonClick,
+    onBookmarkButtonClick
+}: ScenarioActionButtonsProps) {
     return (
         <Grid container>
             <Grid item>
-                <ScenarioOverviewItem action={props.targets.minusButtonTarget}>
+                <ScenarioHeaderIcon onClick={onCollapseButtonClick}>
                     <RemoveIcon fontSize="inherit" />
-                </ScenarioOverviewItem>
+                </ScenarioHeaderIcon>
             </Grid>
             <Grid item>
-                <ScenarioOverviewItem action={props.targets.plusButtonTarget}>
-                    <AddIcon></AddIcon>
-                </ScenarioOverviewItem>
+                <ScenarioHeaderIcon onClick={onExpandButtonClick}>
+                    <AddIcon />
+                </ScenarioHeaderIcon>
             </Grid>
             <Grid item>
-                <ScenarioOverviewItem action={props.targets.printButtonTarget}>
+                <ScenarioHeaderIcon onClick={onPrintButtonClick}>
                     <PrintOutlinedIcon fontSize="inherit" />
-                </ScenarioOverviewItem>
+                </ScenarioHeaderIcon>
             </Grid>
             <Grid item>
-                <ScenarioOverviewItem action={props.targets.bookmarkButtonTarget}>
+                <ScenarioHeaderIcon onClick={onBookmarkButtonClick}>
                     <BookmarkOutlinedIcon fontSize="inherit" />
-                </ScenarioOverviewItem>
+                </ScenarioHeaderIcon>
             </Grid>
         </Grid>
     );
 }
 
-function ScenarioOverviewItem(props: { children: React.ReactNode; action: MouseEventHandler }) {
-    const sx = {
-        width: "12px", // or some other value
-        height: "12px", // same as width
-        p: 0.01,
-        display: "flex",
-        alignItems: "center",
-        justifyContent: "center",
-        border: "1px solid grey",
-        borderRadius: 1,
-        marginRight: "5px"
-    };
+type ScenarioHeaderIconProps = PropsWithChildren<{ onClick: () => void }>;
+
+function ScenarioHeaderIcon({ children, onClick }: ScenarioHeaderIconProps) {
     return (
-        <Box sx={sx} component="span">
-            <Button
-                className="actionPanelButton"
-                color="inherit"
-                sx={{
-                    "&:hover": { textDecoration: "none", color: "inherit" },
-                    "min-width": "1px",
-                    "min-height": "1px",
-                    height: sx.height
-                }}
-                onClick={props.action}
-            >
-                {props.children}
-            </Button>
-        </Box>
+        <StyledIconContainer>
+            <StyledIconButton className="actionPanelButton" onClick={onClick}>
+                {children}
+            </StyledIconButton>
+        </StyledIconContainer>
     );
 }
 
+export enum ScenarioStatusFilter {
```

Comment:
Since this is a status and not a filter I would rename it to `ScenarioFilterStatus`